edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -126,7 +117,8 @@ const CardScreenComponent: ReactComponent<{
         })
     }
 
-    const editCard = () => { router.push({ pathname: `/deck/${deckId}/card/edition` as any, params: { deckId: deckId, prev_question: card?.question, prev_answer: card?.answer, cardIndex: cardIndex } }) }
+    //const editCard = () => { router.push({ pathname: `/deck/${deckId}/card/edition` as any, params: { deckId: deckId, prev_question: card?.question, prev_answer: card?.answer, cardIndex: cardIndex } }) }
```

Comment:
Could this commented code be removed ? For clean code purpose

Reply:
Yep 👍